src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -147,14 +164,15 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
       );
 
       this.compositeClient = compositeClient;
-
+      dd.log('[connectNetwork]: composite client connected successfully', { params: parsedParams });
       try {
         if (nobleValidatorUrl) {
           this.nobleClient = new NobleClient(nobleValidatorUrl);
           if (this.nobleWallet) await this.nobleClient.connect(this.nobleWallet);
+          if (this.nobleClient.isConnected) dd.log('noble client connected successfully');
```

Comment:
I can imagine this log being $$$pammed. Is just catching the errors not enough?

Reply:
yeah fair, gonna just stamp @rosepuppy 's pr https://github.com/dydxprotocol/v4-web/pull/1088#pullrequestreview-2341194540